src/main/java/it/gov/pagopa/pu/pagopapayments/endpoint/PaForNodeEndpoint.java

```diff
@@ -111,7 +117,30 @@ public PaSendRTV2Response paSendRTV2(@RequestPayload PaSendRTV2Request request)
     try {
       log.info("processing paSendRTV2 idPA[{}] notice[{}/{}]", request.getIdPA(), request.getReceipt().getFiscalCode(), request.getReceipt().getNoticeNumber());
       PaSendRtDTO paSendRtDTO = paSendRTMapper.paSendRtV2Request2PaSendRtDTO(request);
-      receiptService.processReceivedReceipt(paSendRtDTO);
+      var registryLoggerResult = registryLogger.execute(
+        paSendRtDTO.getFiscalCode(),
+        paSendRtDTO.getIdBrokerPA(),
+        request.getReceipt().getIdPSP(),
+        request.getReceipt().getIdChannel(),
+        request.getReceipt().getPaymentMethod(),
+        null, // @TODO: no ccp in request
+        RegistryEventType.paSendRTV2,
+        request.getReceipt().getNoticeNumber(),
+        paSendRtDTO,
+        () -> {
+          receiptService.processReceivedReceipt(paSendRtDTO);
+          return Triple.of(null, null, RegistryEventOutcome.OK);
+        },
+        exception -> {
+          log.error("Error processing paSendRTV2 for notice [{}/{}]", paSendRtDTO.getFiscalCode(), paSendRtDTO.getNoticeNumber(), exception);
+          return Triple.of(null, exception, null);
+        }
```

Comment:
this represents the catch already implemented
```suggestion
        exception -> {
          PaSendRTV2Response out;
          if( exception instanceOf PagoPaNodeFaultException spe) {
             log.error("Fault in paSendRTV2 [{}/{}] {}", request.getReceipt().getNoticeNumber(), request.getReceipt().getFiscalCode(), spe.getErrorCode());
             out = handleFault(spe.getErrorCode(), spe.getErrorEmitter(), new PaSendRTV2Response());
          } else {
             log.error("Error in paSendRTV2 [{}/{}] {}", request.getReceipt().getNoticeNumber(), request.getReceipt().getFiscalCode(), request.getReceipt().getReceiptId(), exception);
             out = handleFault(PagoPaNodeFaults.PAA_SYSTEM_ERROR, request.getIdPA(), new PaSendRTV2Response());
          }
          return Triple.of(out, null, RegistryEventOutcome.KO);
        }
```